winlogbeat/docs/modules/sysmon.asciidoc

```diff
@@ -0,0 +1,35 @@
+[[winlogbeat-module-sysmon]]
+[role="xpack"]
+== Sysmon Module
+
+beta[]
+
+The sysmon module processes event log records from the
+https://docs.microsoft.com/en-us/sysinternals/downloads/sysmon[Sysinternals
+System Monitor] (Sysmon) which is a Windows service and device driver that logs
+system activity to the event log. Sysmon is not bundled with Windows or
+Winlogbeat and must be installed independently.
+
+The default configuration file includes configuration for Sysmon. If you do not
+have Sysmon installed Winlogbeat will log a warning that you can ignore stating
+that it could not read from the `Microsoft-Windows-Sysmon/Operational` channel.
```

Comment:
I stumbled over this long sentence. Maybe split it / merge it with the next, e.g.: `If you do not
have Sysmon installed Winlogbeat will log a warning that it could not read from the `Microsoft-Windows-Sysmon/Operational` channel. You can ignore this, it will continue to read from the other configured channels.`

Reply:
To simplify the long sentence I removed the comment about ignoring the warning. I think that's implicit.